### DISCUSSION TITLE
Actions stuck after completion

### DISCUSSION THREAD

**Discussion by dnorth98** [community/community#25590] [Actions]
<p>Odd issue here happening on a lot of actions.  The action completes quickly and then gets stuck just completing.  This is causing other actions to back up (queued) because of these ones not exiting.</p>
<p>Here’s an example.  This one finishes in 6s but it’s been “running” for 43 minutes.</p>
<p><img src="https://user-images.githubusercontent.com/6616703/181091926-a43fef40-f512-46f8-9a69-b7e2690ce496.png" alt="action done.png" data-base62-sha1="nTc8Ik1TGwDGcnhWONAQlhZ8VAv" width="542" height="261"></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/57894856e9c0cd01700e35e976578f3c33982c87.png" data-download-href="/uploads/short-url/cunFjZIxtDmiT9ivL9f6e3jxlAP.png?dl=1" title="action not exiting.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/6616703/181091927-52e51198-a516-4f62-91dd-3e44056230cb.png" alt="action not exiting.png" data-base62-sha1="cunFjZIxtDmiT9ivL9f6e3jxlAP" width="690" height="29" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/57894856e9c0cd01700e35e976578f3c33982c87_2_690x29.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/57894856e9c0cd01700e35e976578f3c33982c87_2_1035x43.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/5/57894856e9c0cd01700e35e976578f3c33982c87_2_1380x58.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/57894856e9c0cd01700e35e976578f3c33982c87_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">action not exiting.png</span><span class="informations">1757×76 9.76 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>It’s like some teardown isn’t running correctly.  These are not using local runners.</p>
<p>Any way to see what is going on?</p>

**Comment by dnorth98** [comment]
<p>Anyone at Github comment on why all our actions seem to be stuck?  Seems like a lot of threads on here today with the same issue.  Any update would be good.</p>

**Comment by SteveShaffer** [comment, marked as answer]
<p>I just got a response back from GitHub support:</p>
<p>Hello -</p>
<p>Thanks for the report! Our engineers have been investigating and have a potential mitigation in place. Builds should start getting unstuck but if that’s not the case for you, please let us know.</p>
<p>Thank you,<br>
Robert</p>

**Comment by dnorth98** [comment]
<p>Wow, thanks <a class="mention" href="/steveshaffer">@steveshaffer</a> !  Appreciate the update</p>

**Comment by Burnett01** [comment]
<p>Thanks Steve!</p>

**Comment by akhimji** [comment]
<p>Steve has anything changed on your end?</p>
<p>We still have multiple jobs in two repos stuck in queued. About to hit 6 hours</p>

**Comment by atifsyedali** [comment]
<p>Same. Some jobs completed and some are still queued for 6 hours+</p>

**Comment by roni-frantchi** [comment]
<p>I’m still seeing this. We’ve been seeing this for a while now - on nodejs jobs it seems</p>
